packages/interbit-cli/scripts/start.js

```diff
@@ -1,87 +1,46 @@
 // © 2018 BTL GROUP LTD -  This package is licensed under the MIT license https://opensource.org/licenses/MIT
 const {
   argOptions,
-  createChains: { createChainsFromConfig },
-  generateDeploymentDetails,
   getConfig,
   getManifest,
   isArg,
   getArg,
   logo,
-  startInterbit,
-  setRootChainManifest,
-  updateIndexHtmls,
-  validateConfig,
-  // watchChain,
-  watchCovenants
+  start
 } = require('interbit')
 
-const startInterbitNode = async () => {
+const run = async () => {
   try {
     console.log(logo)
-
     const options = getOptions(process.argv)
 
-    console.log('Running with options', JSON.stringify(options, null, 2))
-
-    const interbitConfig = getConfig()
-
-    // eslint-disable-next-line
-    const interbitManifest = getManifest()
-    // TODO: Use the manifest as initial variable resolution for generating the new one #262
-
-    validateConfig(interbitConfig)
-
-    const { cli } = await startInterbit(undefined, options)
-    // TODO: This create based on config should only run in dev mode, else run chains based on manifest #276
-    const { chainManifest, covenantHashes } = await createChainsFromConfig(
-      cli,
-      interbitConfig
-    )
-
-    // TODO: This manifest mock is no longer required
-    const deploymentDetails = generateDeploymentDetails(
-      chainManifest,
-      covenantHashes
-    )
-
-    console.log('available chains + covenants: ', deploymentDetails)
-
-    setRootChainManifest(cli, deploymentDetails, interbitConfig)
-
-    if (options.isDevModeEnabled && options.isWatchModeEnabled) {
-      watchCovenants(cli, interbitConfig, chainManifest)
+    if (!options.config) {
+      console.error(
+        'interbit start: Could not find a config file to start with'
+      )
+      process.exit(1)
     }
 
-    if (!options.isDevModeEnabled) {
-      // TODO: We are not in dev mode so output the diff'd manifest
-
-      // TODO: Use the manifest to update index.html instead of "deploymentDetails"
-      updateIndexHtmls({
-        config: interbitConfig, // manifest here
-        chains: deploymentDetails.chains
-      })
-    } else {
-      updateIndexHtmls({
-        config: interbitConfig,
-        chains: deploymentDetails.chains
-      })
+    if (!options.dbPath) {
+      console.warn(
```

Comment:
It is overwritten. I would like to make it so that our users don't worry about the env vars at all.

Since it is still in use as an env by core, however, I'll add another warning.